apis/infix-en_GB/atrium-api-infix-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/infix/en_GB/sequenceAssertions.kt

```diff
@@ -0,0 +1,26 @@
+import ch.tutteli.atrium.creating.AssertionPlant
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl.changeSubject
+
+/**
+ * Turns `Expect<E, T : Sequence<E>>` into `Expect<Iterable<E>>`.
+ *
+ * The transformation as such is not reflected in reporting.
+ * Use `feature(Sequence::asIterable)` if you want to show the transformation in reporting.
+ *
+ * @return The newly created [AssertionPlant] for the transformed subject.
```

Comment:
```suggestion
 * @return The newly created [Expect] for the transformed subject.
```
(Btw. you can submit suggestions also via GitHub)